MapPointMapper/Parser/Parser.swift

```diff
@@ -206,17 +214,17 @@ class Parser {
   }
 
   /**
-  :abstract: Attempt to determine if a given string is in `Well-Known Text` format (GeoString as its referred to internally)
+  _abstract_: Attempt to determine if a given string is in `Well-Known Text` format (GeoString as its referred to internally)
 
-  :discussion: This strips any leading & trailing white space before checking for the existance of word characters at the start of the string.
+  _discussion_: This strips any leading & trailing white space before checking for the existance of word characters at the start of the string.
 
-  :param: input String to attempt determine if is in `Well-Known Text` format
+  - parameter input: String to attempt determine if is in `Well-Known Text` format
 
-  :returns: `true` if it thinks it is, `false` otherwise
+  - returns: `true` if it thinks it is, `false` otherwise
   */
   internal func isProbablyGeoString(input: String) -> Bool {
     let stripped = input.stringByStrippingLeadingAndTrailingWhiteSpace()
-    if let geoString = stripped.rangeOfString("^\\w+", options: .RegularExpressionSearch) {
+    if let _ = stripped.rangeOfString("^\\w+", options: .RegularExpressionSearch) {
```

Comment:
yeah, that works. The migrator put the `_` in for me so I just left it. I think your suggested change & a comment above it would probably be the best. I'd prefer to avoid 'allowing' warnings if at all possible.